APIs/schemas/edid.json

```
@@ -53,13 +54,21 @@
     },
     "display_type": {
       "description": "Digital display type in terms of supported subsampling modes",
-      "type": "string",
-      "enum": [
-        "RGB 4:4:4",
-        "RGB 4:4:4 + YUV 4:4:4",
-        "RGB 4:4:4 + YUV 4:2:2",
-        "RGB 4:4:4 + YUV 4:4:4 + YUV 4:2:2"
-      ]
+      "type": "array",
+      "minItems": 1,
+      "maxItems": 3,
+      "uniqueItems": true,
+      "contains": {
+        "const": "RGB444"
+      },
+      "items": {
+        "type": "string",
+        "enum": [
+          "RGB444",
+          "YUV444",
+          "YUV422"
+        ]
```

Maybe we should match the RX Caps?

From https://github.com/AMWA-TV/nmos-parameter-registers/tree/capabilities/capabilities

> Name: urn:x-nmos:cap:format:color_sampling
Description: Identifies the acceptable color (sub-)sampling mode of a video stream.
Specification: per AMWA BCP-004-01
Type: string (enumerated values as per ST 2110-20 and the IANA Media Type Sub-Parameter Registry for video/raw: Color (sub-)sampling)
Target: (a) SDP attribute 'a=fmtp:' format-specific parameter 'sampling', (b) uncompressed video Flow 'components.bit_depth'
Applicability: AMWA IS-04

We have these possible values https://www.iana.org/assignments/media-type-sub-parameters/media-type-sub-parameters.xhtml#media-type-sub-parameters-15

---

👍 Matching existing Flow and Receiver Caps values wherever possible seems like a win for the ecosystem to me!

---

I agree and hope that YUV and YCbCr are the same in this context. Also, CTA-861 timings maybe should be expanded with color subsampling information.
```suggestion
        "enum": [
          "RGB",
          "YCbCr-4:4:4",
          "YCbCr-4:2:2"
        ]
```